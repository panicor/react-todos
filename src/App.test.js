import React from "react";
import { render  } from '@testing-library/react';
import App from './App';

test('renders', () => {
  render(<App />);
});

test("matches snapshot", function() {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});

