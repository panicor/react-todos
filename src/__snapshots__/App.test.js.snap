// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        <form>
          <label
            for="task"
          >
            Task:
          </label>
          <input
            id="task"
            name="task"
            type="text"
            value=""
          />
          <button>
            Add todo
          </button>
        </form>
      </div>
      <ul />
    </div>
  </div>
</DocumentFragment>
`;
